main.js

```diff
@@ -224,6 +235,7 @@ exports.onUnload = function (reason) {
     if (win.VerticalTabs) {
       win.VerticalTabs.unload();
       let mainWindow = win.document.getElementById('main-window');
+      mainWindow.setAttribute('doNotReverse', 'true');
```

Comment:
I don't think so. As this is only necessary for upgrades.
On upgrade old version will `unload`, then new version will `load`. So long as this piece stays in the `unload` function from now on it will not reorder the tabs. 
If someone uninstalls, then re-installs a later version they may have trouble.